### ISSUE TITLE
Not working with jupyter-server-proxy

### ISSUE THREAD

**Issue by kwunlyou** [tensorflow/tensorboard#5531]
## Environment information (required)

> ### Diagnostics
> 
> <details>
> <summary>Diagnostics output</summary>
> 
> ``````
> --- check: autoidentify
> INFO: diagnose_tensorboard.py version e43767ef2b648d0d5d57c00f38ccbd38390e38da
> 
> --- check: general
> INFO: sys.version_info: sys.version_info(major=3, minor=8, micro=10, releaselevel='final', serial=0)
> INFO: os.name: posix
> INFO: os.uname(): posix.uname_result(sysname='Linux', nodename='*********', release='5.4.72-microsoft-standard-WSL2', version='#1 SMP Wed Oct 28 23:40:43 UTC 2020', machine='x86_64')
> INFO: sys.getwindowsversion(): N/A
> 
> --- check: package_management
> INFO: has conda-meta: False
> INFO: $VIRTUAL_ENV: None
> 
> --- check: installed_packages
> INFO: installed: tensorboard==2.8.0
> INFO: installed: tensorflow==2.7.0
> INFO: installed: tensorflow-estimator==2.7.0
> INFO: installed: tensorboard-data-server==0.6.1
> 
> --- check: tensorboard_python_version
> INFO: tensorboard.version.VERSION: '2.8.0'
> 
> --- check: tensorflow_python_version
> INFO: tensorflow.__version__: '2.7.0'
> INFO: tensorflow.__git_version__: 'v2.7.0-rc1-69-gc256c071bb2'
> 
> --- check: tensorboard_data_server_version
> INFO: data server binary: '/home/kun/.local/lib/python3.8/site-packages/tensorboard_data_server/bin/server'
> INFO: data server binary version: b'rustboard 0.6.1'
> 
> --- check: tensorboard_binary_path
> INFO: which tensorboard: b'/home/kun/.local/bin/tensorboard\n'
> 
> --- check: addrinfos
> socket.has_ipv6 = True
> socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
> socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
> socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 32>
> socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
> Loopback flags: <AddressInfo.AI_ADDRCONFIG: 32>
> Loopback infos: [(<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('127.0.0.1', 0))]
> Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
> Wildcard infos: [(<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('0.0.0.0', 0)), (<AddressFamily.AF_INET6: 10>, <SocketKind.SOCK_STREAM: 1>, 6, '', ('::', 0, 0, 0))]
> 
> --- check: readable_fqdn
> INFO: socket.getfqdn(): '************'
> 
> --- check: stat_tensorboardinfo
> INFO: directory: /tmp/.tensorboard-info
> INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=54451, st_dev=2064, st_nlink=2, st_uid=1000, st_gid=1000, st_size=4096, st_atime=1643115809, st_mtime=1643122711, st_ctime=1643122711)
> INFO: mode: 0o40777
> 
> --- check: source_trees_without_genfiles
> INFO: tensorboard_roots (1): ['/home/kun/.local/lib/python3.8/site-packages']; bad_roots (0): []
> 
> --- check: full_pip_freeze
> INFO: pip freeze --all:
> absl-py==1.0.0
> aiohttp==3.8.1
> aiosignal==1.2.0
> altair==4.1.0
> anyio==3.4.0
> argon2-cffi==21.2.0
> argon2-cffi-bindings==21.2.0
> astor==0.8.1
> astunparse==1.6.3
> async-timeout==4.0.1
> attrs==21.2.0
> awscli==1.22.26
> Babel==2.9.1
> backcall==0.2.0
> backports.zoneinfo==0.2.1
> base58==2.1.1
> black==21.12b0
> bleach==4.1.0
> blinker==1.4
> bokeh==2.4.2
> botocore==1.23.26
> Brotli==1.0.9
> cachetools==4.2.4
> certifi==2021.10.8
> cffi==1.15.0
> charset-normalizer==2.0.9
> click==8.0.3
> colorama==0.4.3
> colorlog==6.6.0
> command-not-found==0.3
> cycler==0.11.0
> dash==2.0.0
> dash-bootstrap-components==1.0.1
> dash-colorscales==0.0.4
> dash-core-components==2.0.0
> dash-daq==0.5.0
> dash-html-components==2.0.0
> dash-table==5.0.0
> debugpy==1.5.1
> decorator==5.1.0
> defusedxml==0.7.1
> docutils==0.15.2
> dtale==1.61.1
> easydev==0.12.0
> entrypoints==0.3
> et-xmlfile==1.1.0
> fitter==1.4.0
> flake8==4.0.1
> Flask==2.0.2
> Flask-Compress==1.10.1
> flask-ngrok==0.0.25
> flatbuffers==2.0
> fonttools==4.28.3
> frozenlist==1.2.0
> future==0.18.2
> gast==0.4.0
> ghp-import==2.0.2
> gitdb==4.0.9
> GitPython==3.1.24
> google-auth==1.35.0
> google-auth-oauthlib==0.4.6
> google-pasta==0.2.0
> grpcio==1.43.0
> h5py==3.6.0
> idna==3.3
> importlib-metadata==4.8.2
> importlib-resources==5.4.0
> ipykernel==6.6.0
> ipython==7.30.1
> ipython-genutils==0.2.0
> ipywidgets==7.6.5
> itsdangerous==2.0.1
> jedi==0.18.1
> Jinja2==3.0.3
> jmespath==0.10.0
> joblib==1.1.0
> json5==0.9.6
> jsonschema==4.2.1
> jupyter==1.0.0
> jupyter-client==7.1.0
> jupyter-console==6.4.0
> jupyter-core==4.9.1
> jupyter-server==1.12.1
> jupyter-server-proxy==3.2.1
> jupyterlab==3.2.4
> jupyterlab-pygments==0.1.2
> jupyterlab-server==2.8.2
> jupyterlab-widgets==1.0.2
> kaleido==0.2.1
> keras==2.7.0
> Keras-Preprocessing==1.1.2
> kiwisolver==1.3.2
> libclang==12.0.0
> lz4==3.1.10
> Markdown==3.3.6
> MarkupSafe==2.0.1
> matplotlib==3.5.0
> matplotlib-inline==0.1.3
> mccabe==0.6.1
> mergedeep==1.3.4
> missingno==0.4.2
> mistune==0.8.4
> mkdocs==1.2.3
> multidict==5.2.0
> mypy-extensions==0.4.3
> nbclassic==0.3.4
> nbclient==0.5.9
> nbconvert==6.3.0
> nbformat==5.1.3
> nest-asyncio==1.5.4
> networkx==2.6.3
> notebook==6.4.7
> numpy==1.21.4
> oauthlib==3.1.1
> openpyxl==3.0.9
> opt-einsum==3.3.0
> packaging==21.3
> pandas==1.3.4
> pandocfilters==1.5.0
> panel==0.12.5
> param==1.12.0
> parso==0.8.3
> pathspec==0.9.0
> patsy==0.5.2
> pexpect==4.8.0
> pickleshare==0.7.5
> Pillow==8.4.0
> pip==21.3.1
> platformdirs==2.4.1
> plotly==5.4.0
> ppscore==1.2.0
> prometheus-client==0.12.0
> prompt-toolkit==3.0.24
> protobuf==3.19.1
> ptyprocess==0.7.0
> pyarrow==6.0.1
> pyasn1==0.4.8
> pyasn1-modules==0.2.1
> pycodestyle==2.8.0
> pycparser==2.21
> pyct==0.4.8
> pydeck==0.7.1
> pyflakes==2.4.0
> Pygments==2.10.0
> Pympler==0.9
> pyparsing==3.0.6
> pyrsistent==0.18.0
> python-dateutil==2.8.2
> pytz==2021.3
> pytz-deprecation-shim==0.1.0.post0
> pyviz-comms==2.1.0
> PyYAML==5.3.1
> pyyaml_env_tag==0.1
> pyzmq==22.3.0
> qtconsole==5.2.1
> QtPy==1.11.3
> requests==2.26.0
> requests-oauthlib==1.3.0
> rsa==4.7.2
> s3transfer==0.5.0
> scikit-learn==0.24.2
> scipy==1.7.3
> seaborn==0.11.2
> Send2Trash==1.8.0
> setuptools==45.2.0
> setuptools-scm==6.3.2
> simpervisor==0.4
> simplejson==3.16.0
> six==1.16.0
> smmap==5.0.0
> sniffio==1.2.0
> squarify==0.4.3
> statsmodels==0.13.1
> streamlit==1.2.0
> strsimpy==0.2.1
> systemd-python==234
> tenacity==8.0.1
> tensorboard==2.8.0
> tensorboard-data-server==0.6.1
> tensorboard-plugin-wit==1.8.1
> tensorflow==2.7.0
> tensorflow-estimator==2.7.0
> tensorflow-io-gcs-filesystem==0.23.1
> termcolor==1.1.0
> terminado==0.12.1
> testpath==0.5.0
> threadpoolctl==3.0.0
> toml==0.10.2
> tomli==1.2.2
> toolz==0.11.2
> tornado==6.1
> tqdm==4.62.3
> traitlets==5.1.1
> typing_extensions==4.0.1
> tzdata==2021.5
> tzlocal==4.1
> ufw==0.36
> urllib3==1.26.7
> validators==0.18.2
> watchdog==2.1.6
> wcwidth==0.2.5
> webencodings==0.5.1
> websocket-client==1.2.1
> Werkzeug==2.0.2
> wheel==0.34.2
> widgetsnbextension==3.5.2
> wrapt==1.13.3
> xarray==0.20.1
> xlrd==2.0.1
> yarl==1.7.2
> zipp==3.6.0
> 
> ``````
> 
> </details>
> 
> ### Next steps
> 
> No action items identified. Please copy ALL of the above output,
> including the lines containing only backticks, into your GitHub issue
> or comment. Be sure to redact any sensitive information.
> 

## Issue description
TensorBoard doesn't work with jupyter-server-proxy. Here are the steps to reproduce:

1. generate `logs` using the example in https://www.tensorflow.org/tensorboard/get_started
2. Run `jupyter-notebook --no-browser --debug`
3. Run ` tensorboard --logdir logs/` and open http://localhost:8888/proxy/6006 but nothing is displayed. Open Dev tools console and see the errors
![image](https://user-images.githubusercontent.com/1352316/151002968-2db78834-23ab-4c7f-a058-5551b55f6bb1.png)
More log information from jupyter as follows

> kun@**********:~/Downloads$ jupyter-notebook --no-browser --debug
> [D 15:05:35.412 NotebookApp] Searching ['/mnt/c/Users/kun.liu/Downloads', '/home/kun/.jupyter', '/home/kun/.local/etc/jupyter', '/usr/etc/jupyter', '/usr/local/etc/jupyter', '/etc/jupyter'] for config files
> [D 15:05:35.412 NotebookApp] Looking for jupyter_config in /etc/jupyter
> [D 15:05:35.412 NotebookApp] Looking for jupyter_config in /usr/local/etc/jupyter
> [D 15:05:35.412 NotebookApp] Looking for jupyter_config in /usr/etc/jupyter
> [D 15:05:35.412 NotebookApp] Looking for jupyter_config in /home/kun/.local/etc/jupyter
> [D 15:05:35.412 NotebookApp] Looking for jupyter_config in /home/kun/.jupyter
> [D 15:05:35.412 NotebookApp] Looking for jupyter_config in /mnt/c/Users/kun.liu/Downloads
> [D 15:05:35.414 NotebookApp] Looking for jupyter_notebook_config in /etc/jupyter
> [D 15:05:35.414 NotebookApp] Looking for jupyter_notebook_config in /usr/local/etc/jupyter
> [D 15:05:35.414 NotebookApp] Looking for jupyter_notebook_config in /usr/etc/jupyter
> [D 15:05:35.414 NotebookApp] Looking for jupyter_notebook_config in /home/kun/.local/etc/jupyter
> [D 15:05:35.414 NotebookApp] Looking for jupyter_notebook_config in /home/kun/.jupyter
> [D 15:05:35.414 NotebookApp] Loaded config file: /home/kun/.jupyter/jupyter_notebook_config.py
> [D 15:05:35.414 NotebookApp] Loaded config file: /home/kun/.jupyter/jupyter_notebook_config.json
> [D 15:05:35.414 NotebookApp] Looking for jupyter_notebook_config in /mnt/c/Users/kun.liu/Downloads
> [D 15:05:35.417 NotebookApp] Raising open file limit: soft 1024->4096; hard 4096->4096
> [D 15:05:35.423 NotebookApp] Paths used for configuration of jupyter_notebook_config:
>         /etc/jupyter/jupyter_notebook_config.json
> [D 15:05:35.423 NotebookApp] Paths used for configuration of jupyter_notebook_config:
>         /usr/local/etc/jupyter/jupyter_notebook_config.json
> [D 15:05:35.423 NotebookApp] Paths used for configuration of jupyter_notebook_config:
>         /usr/etc/jupyter/jupyter_notebook_config.json
> [D 15:05:35.424 NotebookApp] Paths used for configuration of jupyter_notebook_config:
>         /home/kun/.local/etc/jupyter/jupyter_notebook_config.d/jupyter-server-proxy-notebookserverextension.json
>         /home/kun/.local/etc/jupyter/jupyter_notebook_config.d/jupyterlab.json
>         /home/kun/.local/etc/jupyter/jupyter_notebook_config.d/panel-client-jupyter.json
>         /home/kun/.local/etc/jupyter/jupyter_notebook_config.json
> [D 15:05:35.424 NotebookApp] Paths used for configuration of jupyter_notebook_config:
>         /home/kun/.jupyter/jupyter_notebook_config.json
> [I 2022-01-25 15:05:35.951 LabApp] JupyterLab extension loaded from /home/kun/.local/lib/python3.8/site-packages/jupyterlab
> [I 2022-01-25 15:05:35.951 LabApp] JupyterLab application directory is /home/kun/.local/share/jupyter/lab
> [I 15:05:35.955 NotebookApp] Serving notebooks from local directory: /mnt/c/Users/kun.liu/Downloads
> [I 15:05:35.955 NotebookApp] Jupyter Notebook 6.4.7 is running at:
> [I 15:05:35.955 NotebookApp] http://localhost:8888/?token=056a60fb2d328f6662d680554df425a9befe9e2946286290
> [I 15:05:35.955 NotebookApp]  or http://127.0.0.1:8888/?token=056a60fb2d328f6662d680554df425a9befe9e2946286290
> [I 15:05:35.955 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
> [C 15:05:35.958 NotebookApp]
> 
>     To access the notebook, open this file in a browser:
>         file:///home/kun/.local/share/jupyter/runtime/nbserver-7646-open.html
>     Or copy and paste one of these URLs:
>         http://localhost:8888/?token=056a60fb2d328f6662d680554df425a9befe9e2946286290
>      or http://127.0.0.1:8888/?token=056a60fb2d328f6662d680554df425a9befe9e2946286290
> [D 15:05:39.494 NotebookApp] Proxying request to http://localhost:6006/
> [D 15:05:39.498 NotebookApp] 200 GET /proxy/6006 (127.0.0.1) 5.340000ms
> [D 15:05:39.530 NotebookApp] Using contents: services/contents
> [D 15:05:39.564 NotebookApp] Path base/images/favicon.ico served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/base/images/favicon.ico
> [D 15:05:39.565 NotebookApp] Path components/jquery-ui/themes/smoothness/jquery-ui.min.css served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/components/jquery-ui/themes/smoothness/jquery-ui.min.css
> [D 15:05:39.565 NotebookApp] Path components/jquery-typeahead/dist/jquery.typeahead.min.css served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/components/jquery-typeahead/dist/jquery.typeahead.min.css
> [D 15:05:39.566 NotebookApp] Path style/style.min.css served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/style/style.min.css
> [D 15:05:39.567 NotebookApp] Path components/es6-promise/promise.min.js served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/components/es6-promise/promise.min.js
> [D 15:05:39.567 NotebookApp] Path components/react/react.production.min.js served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/components/react/react.production.min.js
> [D 15:05:39.567 NotebookApp] Path components/react/react-dom.production.min.js served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/components/react/react-dom.production.min.js
> [D 15:05:39.568 NotebookApp] Path components/create-react-class/index.js served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/components/create-react-class/index.js
> [D 15:05:39.568 NotebookApp] Path components/requirejs/require.js served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/components/requirejs/require.js
> [D 15:05:39.569 NotebookApp] Path base/images/logo.png served from /home/kun/.local/lib/python3.8/site-packages/notebook/static/base/images/logo.png
> [W 15:05:39.570 NotebookApp] 404 GET /proxy/index.js?_file_hash=4a366cbe (127.0.0.1) 40.850000ms referer=http://localhost:8888/proxy/6006
> [D 15:05:39.605 NotebookApp] Using contents: services/contents
> [W 15:05:39.607 NotebookApp] 404 GET /font-roboto/oMMgfZMQthOryQo9n22dcuvvDin1pK8aKteLpeZ5c0A.woff2 (127.0.0.1) 2.370000ms referer=http://localhost:8888/proxy/6006

**Comment by japie1235813** [comment]
You might miss a trailing slash`http://localhost:8888/proxy/6006/`

The following step works for me:
0. `pip install notebook` 
```
$pip freeze --all | grep jupyterlab
jupyterlab==3.2.8
jupyterlab-pygments==0.1.2
jupyterlab-server==2.10.3
jupyterlab-widgets==1.0.2

```
1.`jupyter notebook` (instead of `jupyter-notebook`) ([reference](https://jupyter.org/install))
(note: `jupyter-notebook --no-browser --debug` also works)
2.`tensorboard --logdir <your_log>`
3. access `<your_domain>:8888/proxy/6006/`
